timewatcher/timewatcher.py

```diff
@@ -59,7 +62,7 @@ def _login(
                 "comp": self.config.company_number,
                 "name": self.config.employee_number,
                 "pw": self.config.employee_password,
-            },
+            }, verify=False,
```

Comment:
Why do we need this? From what iv'e seen the change fix the issue even without disabling this..

Reply:
Sometimes you are behind firewall and you cant verify the host, so that argument will ignore it